streaming/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@mastodon/streaming",
+  "license": "AGPL-3.0-or-later",
+  "description": "Mastodon's Streaming Server",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mastodon/mastodon.git"
+  },
+  "scripts": {
+    "start": "node ./index.js"
+  },
+  "dependencies": {
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "is-docker": "2.2.1",
```

Comment:
This package has a new major out, but it's gone ESM only, so we cannot upgrade it until streaming is ESM only. @renchap we may need to add this to the renovate configuration.

https://github.com/sindresorhus/is-docker/releases/tag/v3.0.0